Dockerfile.tests

```diff
@@ -38,11 +35,11 @@ EXPOSE ${APP_PORT}
 # Hack to bust the cache. Refer to run_tests.sh file.
 ARG CACHEBUST=1
 
-#Running the tests
+# Running the tests
 RUN npm test
 
-#Running the infra tests
-ENTRYPOINT ["/bin/bash", "./cico_tests/run_infra_tests.sh"]
+# Running the infra tests
+ENTRYPOINT ["/bin/sh", "./cico_tests/run_infra_tests.sh"]
```

Comment:
There's actually a problem around this that all scripts have a bash interpreter indicator line in them if so they don't run as expected(basically the shebang line)
Please verify this works for you ... else change the interpreter line in the necessary scripts
A good reference might be https://stackoverflow.com/questions/10376206/what-is-the-preferred-bash-shebang

Reply:
Should I convert to this then : #!/usr/bin/env bash?

Reply:
I felt so but we need to verify before doing that